### ISSUE TITLE
[FEATURE] Adding syntax highlighting to field type of html

### ISSUE THREAD

**Issue by zeshhaan** [PipedreamHQ/pipedream#1126]
I was bit overwhelmed when the html field in `send mail using nodemailer` action didn't had any syntax detection and treated my code as plain string irrespective of whether it is a code comment or actual line of code. I could simply write the markup in an IDE and then paste it back to Pipedream but certain times when i want to make some quick edits i find lost between lines and i go back to IDE where it was saved supposedly. 

So it would be great if you have some sort of detection just like we have currently for `code` part where it detects JS syntax.
This would be a bit more to ask, but it would also be great if it can make itself prettier on save.

I don't know of any other platforms having these feature but since pipedream is more focussed towards developers, we kinda miss that while working inside pipedream. It's fine otherwise as we can go back and forth switching IDE and Pipedream to make changes but would be a great addition and would make our workflow faster.

Thanks you ❤

**Comment by dylburger** [comment]
@zeshhaan just curious - would you prefer to edit workflow code locally, in your IDE, or do you prefer the Pipedream interface? We plan to support local development and continue improving the editor (hopefully to support use cases like this), but I'm curious what your preference is.

**Comment by zeshhaan** [comment]
I would normally start from an IDE. 

But with emails, i don't have a need to write emails from scratch every time. Instead once it's setup, it is considered as a template and just some images and links will be replaced in future. When there arise an edit use-case, i usually copy the code from Pipedream to [alter.email](alter.email) and then make changes there and then go back to Pipedream to paste the updated one. I don't go back to IDE to make changes except that i have a need to create one from scratch.

Apart from this, Pipedream interface crashes when email is large(usually is). Is this something expected?

**Comment by dylburger** [comment]
No, we can look into that. Would you mind sharing your workflow with
***@***.***?

What OS / browser are you using?

**Comment by zeshhaan** [comment]
Yes. Btw, i have updated/edited my previous reply to provide more clarity. 

I'm on macOS/Chrome. I have also had a check with my friend who uses Pipedream on Windows10/Chrome and he has the same issue. This usually happens when the html code is bit lengthy. The whole system gets freezed for some moment.

I can share my workflow but looks like your email is hidden with asterix in your reply above.
